unyt/tests/test_unyt_array.py

```diff
@@ -3006,3 +3006,19 @@ def test_std_recursion():
         a.std(),
         unyt_quantity(20.0, m),
     )
+
+
+def test_code_length_multiplication():
```

Comment:
I think this better captures the general problem
```suggestion
def test_custom_unit_multiplication():
```